spec/requests/locations_controller_spec.rb

```diff
@@ -867,7 +867,18 @@
         end
       end
 
-      it "shows scheme" do
+      it "shows locations with correct data wben the new locations layout feature toggle is enabled" do
+        locations.each do |location|
+          expect(page).to have_content(location.id)
+          expect(page).to have_content(location.postcode)
+          expect(page).to have_content(location.name)
+          expect(page).to have_content(location.status)
+        end
+      end
+
+      it "shows locations with correct data wben the new locations layout feature toggle is disabled" do
+        allow(FeatureToggle).to receive(:location_toggle_enabled?).and_return(false)
+        get "/schemes/#{scheme.id}/locations"
```

Comment:
@kosiakkatrina happy with how I've done this? To avoid duplicating lots of code I just repeated the `get "/schemes/#{scheme.id}/locations"` call.